app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -32,24 +58,31 @@ export default function NewNoteForm({ onClose, refetch }) {
       {({ isSubmitting }) => (
         <Form>
           <Input label="Title" name="title" className="mb-6" />
-          <Textarea label="Description" name="description" rows={8} />
-          <div className="nui-pane__footer nui-pane__footer--absolute">
-            <Button
-              onClick={onClose}
-              label="Cancel"
-              size="large"
-              style="secondary"
-            />
+          <Textarea
+            label="Description"
+            name="description"
+            rows={1}
+            className="mb-6"
+          />
+          <Dropdowns />
```

Comment:
Here, rather than providing `mb-6` to each of the individual blocks, please provide `space-y-6` to the parent `Form` container.